terraform/cloudwatch.tf.json.template.py

```diff
@@ -1,199 +1,27 @@
-                    "statistic": "Minimum",
-                    "threshold": "1.0",
-                    "alarm_description": json.dumps({
-                        "slack_channel": "azul-dev",
-                        "environment": config.deployment_stage,
-                        "description": f"azul-{config.deployment_stage} HealthCheckStatus alarm"
-                    }),
-                    "alarm_actions": [
-                        f"arn:aws:sns:{aws.region_name}:{aws.account}:cloudwatch-alarms",
-                        f"arn:aws:sns:{aws.region_name}:{aws.account}:dcp-events"
-                    ],
-                    "ok_actions": [
-                        f"arn:aws:sns:{aws.region_name}:{aws.account}:cloudwatch-alarms",
-                        f"arn:aws:sns:{aws.region_name}:{aws.account}:dcp-events"
-                    ],
-                    "dimensions": {
-                        "HealthCheckId": "${aws_route53_health_check.composite-azul.id}",
-                    }
-                },
-                "data_portal_health": {
-                    "alarm_name": f"data-browser-{config.deployment_stage}",
-                    "actions_enabled": True,
-                    "comparison_operator": "LessThanThreshold",
-                    "evaluation_periods": "1",
-                    "metric_name": "HealthCheckStatus",
-                    "namespace": "AWS/Route53",
-                    "period": "120",
-                    "statistic": "Minimum",
-                    "threshold": "1.0",
-                    "alarm_description": json.dumps({
-                        "slack_channel": "data-browser",
-                        "environment": config.deployment_stage,
-                        "description": f"data-browser-{config.deployment_stage} HealthCheckStatus alarm"
-                    }),
-                    "alarm_actions": [
-                        f"arn:aws:sns:{aws.region_name}:{aws.account}:cloudwatch-alarms",
-                        f"arn:aws:sns:{aws.region_name}:{aws.account}:dcp-events"
-                    ],
-                    "ok_actions": [
-                        f"arn:aws:sns:{aws.region_name}:{aws.account}:cloudwatch-alarms",
-                        f"arn:aws:sns:{aws.region_name}:{aws.account}:dcp-events"
-                    ],
-                    "dimensions": {
-                        "HealthCheckId": "${aws_route53_health_check.composite-portal.id}",
+            "aws_cloudwatch_log_metric_filter": {
+                f"azul-{lambda_}-5xx": {
+                    "name": config.qualified_resource_name(lambda_ + '_5xx'),
+                    "pattern": "{ $.status = 5* }",
+                    "log_group_name": "${aws_cloudwatch_log_group.%s.name}" % lambda_,
+                    "metric_transformation": {
+                        "name": config.qualified_resource_name(lambda_ + '_5xx'),
+                        "namespace": f"azul-{config.deployment_stage}",
```

Comment:
```suggestion
                        "namespace": f"{config.resource_prefix}-{config.deployment_stage}",
```

Reply:
But to simplify things and avoid duplicating the separator

```suggestion
                        "namespace": config.qualified_resource_name('metrics'),
```

even though `metrics` could be implied. Just in general, our goal is to encapsulate our naming convention in functions/methods because it will make it easier to change in the future, like swapping stage and resource_name.